docs/machine-learning/how-to-guides/getting-started-dataframe.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Getting started with DataFrames
+description: Learn how to use DataFrame to manipulate and prepare data.
+author: beccam
```

Comment:
What is the author name here?

Reply:
author is your GH username so this looks right to me.